airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -458,7 +458,7 @@ def create_default_error_handler(self, model: DefaultErrorHandlerModel, config:
             parameters=model.parameters,
         )
 
-    def create_default_paginator(self, model: DefaultPaginatorModel, config: Config, **kwargs) -> DefaultPaginator:
+    def create_default_paginator(self, model: DefaultPaginatorModel, config: Config, *, url_base: str) -> DefaultPaginator:
```

Comment:
I was trying to figure out how this was working when you weren't passing the `url_base` into this function from `create_simple_retriever()`, but I realized that I think I accidentally left the code to pass `url_base` downward in on a prior PR. Just noting it just in case any one else reviewing was confused too.

Reply:
Yes. I preferred to do explicit it since the error message is better that way